.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "singleQuote": true,
+  "parser": "flow",
+  "printWidth": 120,
+  "trailingComma": "none",
```

Comment:
Your eslint config was complaining about trailing commas, so I've switched it off here